Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# proof of concept

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Implemented the way to create black market vehicle list similar to how forbiddenItems are created.
Allows to create addon dependent lists of blackmarket vehicles.
At the moment it's just a proof of concept.    

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
1. Load with RHS GREF loaded and check black trader vehicles
2. Load CUP and check black trader vehicles
3. Try adjusting war level and amount of airfields/military bases controlled to meet confitions for differnt vehicles.

********************************************************
Notes:

Check for messages in .rpt file
